CHANGELOG.mdx

```diff
@@ -16,6 +16,20 @@ https://github.com/RasaHQ/rasa/tree/master/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [1.10.15] - 2020-10-09
```

Comment:
This should probably go under 2.0, but also we're missing the 1.13 and 1.14 changelogs 🤔

Reply:
the changes are not on master anymore, I guess we should can recreate them from our release notes

Reply:
cool, thanks! yeah i see some got in here: https://github.com/RasaHQ/rasa/pull/6942/files